mithril-stm/src/merkle_tree/mod.rs

```diff
@@ -0,0 +1,41 @@
+//! Merkle tree implementation for STM
+
+mod leaf;
+pub use leaf::MTLeaf;
+
+mod tree;
+pub use tree::MerkleTree;
+
+mod commitment;
+pub use commitment::{MerkleTreeCommitment, MerkleTreeCommitmentBatchCompat};
+
+mod path;
+pub use path::{BatchPath, Path};
```

Comment:
This would be more readable:
```suggestion
mod commitment;
mod leaf;
mod path;
mod tree;

pub use commitment::{MerkleTreeCommitment, MerkleTreeCommitmentBatchCompat};
pub use leaf::MTLeaf;
pub use path::{BatchPath, Path};
pub use tree::MerkleTree;
```